### PR TITLE
DAOS-623 build: Update clang-format settings.

### PR DESCRIPTION
**Pull request by ashleypittman**
Change Alignment to be closer to our established code.
Change spaces to tabs to avoid a clang bug.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

**Review by jolivier23 — COMMENTED** [review]
Looks good to me aside from the spelling error.

**Comment by ashleypittman** [comment]
> Looks good to me aside from the spelling error.

Rebased with master and corrected.

File: site_scons/site_tools/extra/extra.py [hunk 1]
```diff
@@ -47,7 +53,31 @@ def _supports_custom_format(clang_exe):
...
+    match = re.search(r'version ([\d+\.]+) ', output)
+    if match:
+        parts = match.group(1).split('.')
+        if int(parts[0]) < 14:
+            return False
+        if int(parts[1]) > 0:
+            return True
+        if int(parts[2]) < 5:
+            return False
+        return True
```
Comment (daltonbohning) [inline site_scons/site_tools/extra/extra.py:79]: FYI, there is a function for this
```
from packaging import version
version.parse('14.0.5') <= version.parse('14.0.4')
False
version.parse('14') <= version.parse('14.0.5')
True
```

**Review by daltonbohning — DISMISSED** [review]

File: site_scons/site_tools/extra/extra.py (hunk 1 above)
Comment (ashleypittman) [reply site_scons/site_tools/extra/extra.py:79]: Awesome, will update.  This wasn't great code.

Comment (ashleypittman) [reply site_scons/site_tools/extra/extra.py:79]: Nice, but I'm not prepared to modify the rpm specfile for just this change.

Comment (daltonbohning) [reply site_scons/site_tools/extra/extra.py:79]: Sorry, I thought that module was built-in :(

**Review by jolivier23 — APPROVED** [review]

**Comment by jolivier23** [comment]
when this is landed, probably needs an announcement of some sort with instructions for enabling it.

**Comment by ashleypittman** [comment]
> when this is landed, probably needs an announcement of some sort with instructions for enabling it.

Possibly better to have two or three people test it for a couple of weeks, and then work on the instructions and announcement.

**Pull request merged**